core/src/test/java/com/linecorp/armeria/server/file/HttpFileTest.java

```diff
@@ -73,4 +80,41 @@ void redirect() throws Exception {
         assertThat(agg.status()).isEqualTo(HttpStatus.TEMPORARY_REDIRECT);
         assertThat(agg.headers().get(HttpHeaderNames.LOCATION)).isEqualTo("/foo/bar?a=b");
     }
+
+    @Test
+    void createFromFileUrl(@TempDir Path tempDir) throws Exception {
+        final URL file = tempDir.resolve("test.txt").toUri().toURL();
+        final HttpFileBuilder builder = HttpFile.builder(file);
+        assertThat(builder).isInstanceOf(HttpFileBuilder.FileSystemHttpFileBuilder.class);
+    }
+
+    @Test
+    void createFromHttpUrl() throws Exception {
+        final URL url = new URL("https://line.me");
+        assertThatThrownBy(() -> HttpFile.builder(url)).isInstanceOf(IllegalArgumentException.class);
```

Comment:
Could add `hasMessageContaining()` check for the exception?

Reply:
Done